multirange.js

```diff
@@ -61,38 +63,41 @@ var multirange = function(input) {
 		ghost.oninput = input.oninput.bind(input);
 	}
 
-	function update() {
+	function update(m) {
```

Comment:
Maybe a more descriptive name? What is m? w and h are single letter because they're rather obvious, not sure m (middle?) is.

Reply:
I mean "mode"